cheroot/server.py

```diff
@@ -1037,8 +1038,8 @@ def read_request_headers(self):
             # Don't use simple_response here, because it emits headers
             # we don't want. See
             # https://github.com/cherrypy/cherrypy/issues/951
-            msg = self.server.protocol.encode('ascii')
-            msg += b' 100 Continue\r\n\r\n'
+            msg = b''.join([self.server.protocol.encode('ascii'),
+                           b' 100 Contunue\r\n\r\n'])
```

Comment:
But better use string interpolation because it would be better readable this way.
Like 
```python
b'{} {}'.format(
    self.server.protocol.encode('ascii'),
    b' 100 Continue\r\n\r\n',
)
```